### PR TITLE
Task6

### PR DESCRIPTION
**Pull request by yiyayou**
turning in task6 for session03

### CODE REVIEW

File: students/YIGU/session03/list_lab.py [hunk 1]
```diff
@@ -0,0 +1,89 @@
+# list_lab.py
```
Comment (am0k) [inline students/YIGU/session03/list_lab.py]: Missing the hash-bang the assignment asked you to put here.

File: students/YIGU/session03/list_lab.py [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+fruit_basket.append(new_fruit)
+
+# display fruit from fruit basket
+print u"Your fruit basket has %s" % fruit_basket
+
+# ask user for a number
+fruit_number = raw_input(u'Please enter a number between 1-5-->')
+
+# display fruit from fruit basket
+print u"Number %s fruit out of your Your fruit basket is %s" % (fruit_number, fruit_basket[fruit_number-1])
```
Comment (am0k) [inline students/YIGU/session03/list_lab.py]: Code lines should not be longer than 79 characters.

Here's the PEP 8 section describing the reasoning behind having line length limits:
https://www.python.org/dev/peps/pep-0008/#maximum-line-length

Short version: it makes code easier for humans to read, and it's a standard that has been set.

File: students/YIGU/session03/list_lab.py [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+
+# Add another fruit to the beginning of the list using “+” and display the list.
+fruit_basket = [u"Mangos"] + fruit_basket
+
+# Add another fruit to the beginning of the list using insert() and display the list.
```
Comment (am0k) [inline students/YIGU/session03/list_lab.py]: This line is longer than 79 characters as well.

File: students/YIGU/session03/list_lab.py [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+# display fruit from fruit basket
+print u"Your fruit basket has %s" % fruit_basket
+
+# Remove the last fruit from the list.
+fruit_basket.pop(len(fruit_basket)-1)
+
+# Display the list.
+print u"Your fruit basket has %s" % fruit_basket
+
+# Ask the user for a fruit to delete and find it and delete it.(Bonus: Multiply the list times two. Keep asking until a match is found. Once found, delete all occurrences.)
```
Comment (am0k) [inline students/YIGU/session03/list_lab.py]: This line is definitively longer than 79 characters.

File: students/YIGU/session03/list_lab.py [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+        if fruit.lower() == delete_fruit.lower():
+            fruit_basket.remove(fruit)
+
+while len(fruit_basket) == number_of_fruit:
+    delete_fruit_function()
+
+#  Ask the user for input displaying a line like “Do you like apples?”
+#  for each fruit in the list (making the fruit all lowercase).
+#  For each “no”, delete that fruit from the list.
+#  For any answer that is not “yes” or “no”, prompt the user to answer with one of those two values (a while loop is good here):
```
Comment (am0k) [inline students/YIGU/session03/list_lab.py]: This line is longer than 79 characters as well.

File: students/YIGU/session03/list_lab.py [hunk 6]
```diff
@@ -0,0 +1,89 @@
...
+reverseletters_fruit_basket = []
+
+for fruit in fruit_basket:
+    reverseletters_fruit = ""
+    for letter in fruit:
+        reverseletters_fruit = letter+reverseletters_fruit
+    reverseletters_fruit_basket.append(reverseletters_fruit)
+
+# Delete the last item of the original list. Display the original list and the copy.
+fruit_basket.remove(len(fruit_basket-1))
```
Comment (am0k) [inline students/YIGU/session03/list_lab.py]: You should leave one empty space to either side of "-".

**Comment by am0k** [comment]
I can't accept the pull request until you Fix your student directory name to "YiGu". :)

**Comment by ppham** [comment]
Hi Yi, please fix the line lengths above. Thanks.

**Pull request merged**